PocketKit/Sources/PocketKit/Article/ReadableViewModel/ReadableEvent.swift

```diff
@@ -2,6 +2,7 @@ import Sync
 
 enum ReadableEvent {
     case archive
+    case moveToSaves
```

Comment:
Both 'moveToSaves' and 'unarchive' are used interchangeably in the PR. Consider sticking to one (personally I'd pick unarchive as it has little chance of being confused for saving a new article)

Reply:
haha agreed! even had a battle with myself when to use which one. I'll keep existing as is, but new changes to use unarchive.